client/src/pages/TestPage/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react'
+import { ProductList } from '@/components/common/ProductList'
+import { client } from '@/ApolloClient'
+import { GET_PRODUCTS } from './gql'
+
+export const TestPage: React.FC = () => {
+  const [products, setProducts] = useState([])
+
+  useEffect(() => {
+    const fetchProducts = async () => {
+      const {
+        data: { getProducts },
+      } = await client.query({ query: GET_PRODUCTS })
+
+      setProducts(getProducts.slice(0, 30))
+    }
+    fetchProducts()
+  }, [])
```

Comment:
```suggestion
const { loading, error, data } = useQuery(GET_PRODUCTS, {
    fetchPolicy: "cache-and-network"
  })
```
이렇게 하면 더 깔끔할 거 같아요!

useQuery가 컴포넌트가 렌더링 될때마다 실행하는 거라 작성하신 코드랑 하는 기능은 똑같을 거에요!

Reply:
확실히 useQuery 사용하는게 깔끌하네요!

다른 브랜치로 옮겨서 작업하는 중이라 useQuery 적용 이슈 새로 파서 한번에 고칠게요!!